src/DoctrineAuditBundle/AuditConfiguration.php

```diff
@@ -128,6 +124,7 @@ public function isPre43Dispatcher(): bool
      */
     public function setEntities(array $entities): void
     {
+        $this->annotationLoaded = true;
```

Comment:
This does not seem logical here: we only set entities in this setter so nothing about annotation loading but we set the annotation loaded flag.

Reply:
its break some tests, becaues annotation loaded on __constructor and after it call setEntities(), so i guess, that setEntities is manualy setter entities without loading annotation.

Reply:
You're right, using this setter currently does not load annotations and that's a design choice: the setter **overrides** config and annotations.

So I think we can keep your `$this->annotationLoaded` here and we need to add a clear comment in the method dockblock indicating that `AuditConfiguration::setEntities()` completely **overrides** default config and annotations.